release_notes/Service_Fabric_ReleaseNotes_10.md

```diff
@@ -0,0 +1,76 @@
+|**Windows - 10.0.1816.9590<br>Ubuntu 18 - 10.0.1728.1<br>Ubuntu 20 - 10.0.1728.1** | **Bug** | Backup Restore Service(BRS) | **Brief Description**: Presently, in the process of deleting expired backups from the backup store, Backup Restore Service (BRS) attempts to retrieve all backups before initiating the deletion of expired backups. However, this approach can encounter repeated failures if the Backup Store is facing pressure and unable to provide all backups to BRS at once. Consequently, this situation prevents BRS from deleting any expired backups, resulting in a continuous accumulation of backup files.  <br>**Solution**: BRS has been updated to perform the removal of expired backups in batches. The process involves the sequential processing of backups within distinct batches housed in the Backup Store. Within each batch, expired backups will be deleted prior to advancing to the subsequent batch. <br> 
+
+
+## Retirement and Deprecation Path Callouts
+* As aligned with [Microsoft .NET and .NET Core - Microsoft Lifecycle | Microsoft Learn](https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-and-net-core), SF Runtime has dropped support for Net Core 3.1 as of December 2022. For  supported versions see [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#current-versions-1) and migrate applications as needed.
+* Ubuntu 18.04 LTS will reach its 5-year end-of-life window in June, 2023. Service Fabric runtime will drop support for 18.04 LTS after the published date, and we recommend moving your clusters and applications to supported versions listed here: [Service Fabric supported Linux versions](https://learn.microsoft.com/en-us/azure/service-fabric/service-fabric-versions#supported-linux-versions-and-support-end-date)
+* Service Fabric runtime will soon stop supporting BinaryFormatter based remoting exception serialization by default and move to using Data Contract Serialization (DCS) based remoting exception serialization by default. Current applications using it will continue to work as-is, but Service Fabric strongly recommends moving to using Data Contract Serialization (DCS) based remoting exception instead.
+* As called out in the previous releases, Service Fabric runtime will remove Service Fabric runtime version 6.4 packages and older, SDK version 3.3 packages and older from the package Download Center in July 2023. 
```

Comment:
Will the package removal affect nuget packages?

Reply:
No.